streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/TaskMetricsTest.java

```diff
@@ -356,4 +354,27 @@ public void shouldGetDroppedRecordsSensorOrLateRecordDropSensor() {
             shouldGetDroppedRecordsSensor();
         }
     }
+
+    @Test
+    public void shouldGetRecordE2ELatencySensor() {
+        final String operation = "record-e2e-latency";
+        expect(streamsMetrics.taskLevelSensor(THREAD_ID, TASK_ID, operation, RecordingLevel.INFO))
+            .andReturn(expectedSensor);
+        expect(streamsMetrics.taskLevelTagMap(THREAD_ID, TASK_ID)).andReturn(tagMap);
+        StreamsMetricsImpl.addMaxAndMinToSensor(
+            expectedSensor,
+            TASK_LEVEL_GROUP,
+            tagMap,
+            operation,
+            RECORD_E2E_LATENCY_MAX_DESCRIPTION,
```

Comment:
req: Please do not use the constant here. The point of this test is also to check the correctness of the description, i.e., the content of that constant.